### ISSUE TITLE
Process fails with `--init-rule-docs` when no docs need to be creates

### ISSUE THREAD

**Issue by TheSonOfThomp** [bmish/eslint-doc-generator#494]
Running `eslint-doc-generator --init-rule-docs` will exit with the following error if no docs need to be created. I would expect this to be skipped in the case where nothing needs to be created.

```sh
$ eslint-doc-generator --init-rule-docs
--init-rule-docs was enabled, but no rule doc file needed to be created.
error Command failed with exit code 1.
```

My idea workflow would be to always pass this flag in case there's been a rule that's been missed

**Comment by bmish** [comment]
This is an interesting point. I hadn't considered that one might want to leave this option in place indefinitely. The original idea was that this option would only be used only temporarily when intentionally creating a new rule doc, but I'm open to removing the exception so it can be used permanently.
